rfm_examples/XBDP_TDD_Example.m

```diff
@@ -0,0 +1,127 @@
+%% XBDP_TDD_Example.m 
+% Description: This script is to be used with the Analog Devices X-Band 
+% Development Platform to demonstrate MATLAB control of the system. 
+% It allows the user to configure the Rx/Tx aspects of the system, then 
+% excercise the TDD core for pulsed radar evaluations. 
+% This script requires the use of the Analog Devices, Inc. High
+% Speed Converter Toolbox and RF & Microwave Toolbox
+%
+% Author: Sam Ringwood
+% Date: February 2023
+
+% Gain Access to the Analog Devices, Inc. High Speed Converter Toolbox at:
+% https://github.com/analogdevicesinc/HighSpeedConverterToolbox
+
+% Gain Access to the Analog Devices, Inc. RF & Microwave Toolbox at:
+% https://github.com/analogdevicesinc/RFMicrowaveToolbox
+
+clear all, close all, clc
+%% Array Mapping
+%verify element maps correctly to hardware!
+subarray = ...
+    [[1 2 5 6 9 10 13 14];... %subarray 1
+    [3 4 7 8 11 12 15 16];... %subarray 2
+    [19 20 23 24 27 28 31 32];... %subarray 3
+    [17 18 21 22 25 26 29 30]]';  %subarray 4 
+subarray_ref = [2 4 18 20]; %subarray reference elements
+dac_map = [4 3 2 1]; %DAC map to subarray
+dac_ref = 4; %DAC reference channel
+adc_map = [4 2 1 3]; %ADC map to subarray
+adc_ref = 4; %ADC reference channel
+
+%% Config Dev Platform
+
+uri = 'ip:192.168.1.211';
+
+interFreq = 4.5e9; %Tx NCO Frequency & Unfolded Rx NCO Frequency [Hz]
+
+fs_RxTxIQ = 250e6; %IQ Sample Rate [Hz]
+periods = 32; %Desired Number Of Periods For Tx Signal
+basebandFreq = fs_RxTxIQ/periods; %Baseband Frequency [Hz]
+
+%Setup AD9081 Tx & Rx
+tx = adi.AD9081.Tx;
+rx = adi.AD9081.Rx;
+tx.uri = uri;
```

Comment:
RX is not used in this script. Is there something missing?

Reply:
Updated the file.